src/data/content/design-system/fondamenti/colori.yaml

```diff
@@ -0,0 +1,954 @@
+# ————————————————————————————————————————————
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  template:
+    name: design-system-index
+
+seo:
+  name: Colori
+  description: "Il sistema di colori del design system della Pubblica Amministrazione italiana, per garantire coerenza visiva nei siti e servizi pubblici digitali"
```

Comment:
`subtitle` e `description` (riga 36) sono coerenti in tutte le altre pagine del sito @danielaiozzo no?

Reply:
yes avete ragione, usiamo la stessa